docs/resources/ces_alarmrule.md

```diff
@@ -109,26 +110,28 @@ and ok_actions coexist, their corresponding notification_list must be of the **s
 
 The `metric` block supports:
 
-* `namespace` - (Required, String) Specifies the namespace in **service.item** format. service.item can be a string of 3
-  to 32 characters that must start with a letter and can consists of uppercase letters, lowercase letters, numbers, or
-  underscores (_). For details,
-  see [Services Interconnected with Cloud Eye](https://support.huaweicloud.com/intl/en-us/api-ces/ces_03_0059.html).
+* `namespace` - (Required, String, ForceNew) Specifies the namespace in **service.item** format. service.item can be a
+  string of 3 to 32 characters that must start with a letter and can consists of uppercase letters, lowercase letters,
+  numbers, or underscores (_). Changing this creates a new resource.
```

Comment:
I suggest to using `The namespace can be a string of 3 to 32 characters that must start with a letter and can consists of letters, numbers, or underscores (_).` to make it the same as the validation message.
Because the description of `uppercase, lowercase` is too redundant.